index-rna.md

```diff
@@ -8,47 +8,51 @@ website: https://rna.usegalaxy.eu
         <div class="col-lg-12">
             <div class="rna-intro-message">
                 <h1>Welcome to <bold>RNA Galaxy</bold></h1>
-                <h3>The <a href="http://bgruening.github.io/galaxy-rna-workbench" target="_blank">RNA workbench</a> developed by the de.NBI RNA Bioinformatics Center</h3>
+                <h3>The <a href="http://bgruening.github.io/galaxy-rna-workbench" target="_top">RNA workbench</a> developed by the de.NBI RNA Bioinformatics Center</h3>
```

Comment:
this should probably stay blank? So open in a new tab?

Reply:
Sorry, that was not intended. Reverted.